src/mame/drivers/tiamc1.cpp

```diff
@@ -331,7 +331,7 @@ static MACHINE_CONFIG_START( tiamc1, tiamc1_state )
 
 	/* video hardware */
 	MCFG_SCREEN_ADD("screen", RASTER)
-	MCFG_SCREEN_RAW_PARAMS(PIXEL_CLOCK, 336, 0, 256, 312, 0, 256)       // pixel clock and htotal comes from docs/schematics, the rest is guess (determined by undumped PROM)
+	MCFG_SCREEN_RAW_PARAMS(PIXEL_CLOCK, 336, 0, 256, 312, 0, 256)       /* pixel clock and htotal come from docs/schematics, the rest is a guesswork (originall specified by an undumped PROM) */
```

Comment:
It should be "the rest is a guess" or "the rest is guesswork".  "the rest is a guesswork" is bad grammar.  Also, "originall" is a typo.